.github/workflows/build-test-package.yml

```diff
@@ -200,7 +200,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        python-version: [37, 38, 39]
+        python-version: [37, 38, 39, 310]
```

Comment:
Maybe don't add 3.10 to the list now. Or even better, fix the issue with Python 3.10 package generation 😄

Reply:
Python 3.10 CI fails at the SlicerExecutionModel build stage, perhaps the issue lies there?

Reply:
Then don't add 3.10 to the list now. Somebody else will deal with that later.